ex_webrtc/lib/ex_webrtc_endpoint.ex

```diff
@@ -125,8 +125,7 @@ defmodule Membrane.RTC.Engine.Endpoint.ExWebRTC do
       bin_input(pad)
       |> child({:track_receiver, track_id}, %TrackReceiver{
         track: track.engine_track,
-        initial_target_variant: :h,
-        keyframe_request_interval: Membrane.Time.seconds(5)
+        initial_target_variant: :h
```

Comment:
I think it would be beneficial to know why it was used in the first place. And also we should check if the first request was made immediately or after `request_interval` value.

Reply:
I put the interval before, just to be on the safe side in case the PLI's aren't propagated for some reason. Now it seems it's not needed anymore.

Reply:
The first request has been made after the `request_interval` time